src/ol/render/canvas/canvasreplay.js

```diff
@@ -1376,6 +1402,16 @@ ol.render.canvas.PolygonReplay.prototype.getBufferedMaxExtent = function() {
 
 
 /**
+ * @param {boolean} rightHanded Whether polygons are assumed to follow
+ *     the right-hand rule.
+ */
+ol.render.canvas.PolygonReplay.prototype.setRightHanded =
```

Comment:
Geometries can come from many places. We could transform feature geometries when we add features to the source. But we cannot enforce that when users modify geometries, use a replay in a postrender event, etc.

This has always been in the renderer code: `getOrientedFlatCoordinates` is used, and the oriented coordinates are cached.